README.md

```diff
@@ -0,0 +1,18 @@
+# Symfony Training
+
+## Installation
+
+```bash
+cp .env.dist .env
+docker-compose build
+docker-compose run --rm php composer install
+docker-compose up -d
```

Comment:
I would suggest you to add a small Makefile to deal with installation steps in a single command, in order to be sure that everyone will execute all the commands.
```
.PHONY: start
start: cp-env build install-dependencies
	docker-compose up -d

.PHONY: cp-env
cp-env:
	cp -n .env.dist .env

.PHONY: build
build:
	docker-compose build

.PHONY: install-dependencies
install-dependencies:
	docker-compose run --rm php composer install
```
Your students will only have to run `make start`

Reply:
I understand the point but I suggest to not use a Makefile so everybody will understand each step to setup the stack.

> Also, [Taskfile](https://taskfile.dev/) is an alternative to consider :wink: I can add a topic on the README to install it with your suggested shotcut?

Reply:
IMO we should stick with a standard Makefile. The goal is to let any knpeer or trainee to install the project quickly with no any dependency but docker. If we require the taskfile CLI to be installed I think it could become an issue for some trainees who are not root on their machine and just cannot install any package on it.

Reply:
> I understand the point but I suggest to not use a Makefile so everybody will understand each step to setup the stack.

I suggested that because IMHO the goal of this training is not to learn how to bootstrap a project with an opinionated stack but only to be able to bootstrap it fast and without issues, so that your trainees will be able to start fast :wink:

Reply:
Ok I'll check if all works fine on a fresh install of Docker from the APT repository and add a Makefile. If all is ok i'll merge the PR